packages/sdk/src/sdk/services/Solana/programs/ClaimableTokensClient/ClaimableTokensClient.ts

```diff
@@ -152,18 +152,22 @@ export class ClaimableTokensClient {
             userBank,
             programId: this.programId
           })
-        const confirmationStrategyArgs =
+        const computeBudgetLimitInstruction =
+          ComputeBudgetProgram.setComputeUnitLimit({
+            units: 50000
```

Comment:
What criteria determines this value? And I thought we used constants that were settable by remote config for things like this?

Reply:
I looked at what the simulation estimated the compute limit to be, and then doubled it. It's a bit of a magic number, can probably revisit later. Want to get this in though because the smaller we can make this number (it defaults to 200k) the more likely validators will let us in their block.